lib/npack.js

```diff
@@ -187,34 +249,26 @@ exports.install = function(options, callback) {
 				}
 			}
 
-			// check node_modules existence in root path
-			fse.pathExists(currentNodeModulesPath, this.slot());
+			logger.log('Sync temp package npm dependencies');
 
-			// check node_modules existence in new package path
-			fse.pathExists(tmpPkgNodeModulesPath, this.slot());
+			// sync node_modules
+			if (options.syncMode === 'install') {
+				syncByInstall({
+					tmpPkgPath: tmpPkgPath,
+					currentPkgPath: options.dir,
+					logger: logger,
+					log: options.log
+				}, this.slot());
+			} else if (options.syncMode === 'ci') {
+				syncByCi({
+					tmpPkgPath: tmpPkgPath,
+					log: options.log
+				}, this.slot());
+			}
```

Comment:
should we throw error in final else? with unknown sync mode

Reply:
I move `syncMode` option validation to `lib/npack.js` so it's not impossible now to have another `syncMode` value here for else branch